niviz/interfaces/views.py

```diff
@@ -48,3 +50,75 @@ def _post_run_hook(self, runtime):
 
     def _run_interface(self, runtime):
         return runtime
+
+
+class _ISegInputSpecRPT(nrc._SVGReportCapableInputSpec,
+                        BaseInterfaceInputSpec):
```

Comment:
turns out nrc._SVGReportCapableInputSpec ends up basing `BaseInterfaceInputSpec` so it isn't needed!
```suggestion
class _ISegInputSpecRPT(nrc._SVGReportCapableInputSpec):
```